test/fork/migration-v2.js

```diff
@@ -665,17 +697,16 @@ describe('V2 upgrade', function () {
     // expect(tcNxmBalance).to.be.equal(rewardsSum.add(coverNotesSum));
   });
 
-  it('remove CR, CD, IC, QD, QT, TF, TD, P2', async function () {
-    await submitGovernanceProposal(
-      PROPOSAL_CATEGORIES.removeContracts, // removeContracts(bytes2[])
-      defaultAbiCoder.encode(['bytes2[]'], [['CR', 'CD', 'IC', 'QD', 'QT', 'TF', 'TD', 'P2'].map(x => toUtf8Bytes(x))]),
-      this.abMembers,
-      this.governance,
-    );
+  // TODO review
+  it('run populate-v2-products script', async function () {
+    await populateV2Products(this.cover.address, this.abMembers[0]);
   });
 
-  it.skip('run populate-v2-products script', async function () {
-    await populateV2Products(this.cover.address, this.abMembers[0]);
+  it('Call LegacyPooledStaking.pushRewards for all non-deprecated contracts', async function () {
+    const productAddresses = PRODUCT_ADDRESSES_OUTPUT.map(address => address.toLowerCase());
+
+    console.log(`Call pushRewards with ${productAddresses.length} product addresses.`);
+    await this.pooledStaking.pushRewards(productAddresses);
```

Comment:
Do you see any issue with calling pushRewards here after

* upgrading in bulk V1 contracts to V2
*calling LegacyPooledStaking.blockV1()

there's no code path to further call pushBurn or accumulateRewards post-upgrade.

Executing this before processPendingActions prepares everything for that.

@roxdanila @shark0der

Reply:
Do we have a ticket for what remains to be done to make pushRewards work well if we do this mid-week?